### PR TITLE
Fix args alias

### PR DESCRIPTION
**Pull request by jusschwitalla**
I have tried to run QUARK with a modules configuration containing devices like that: 
..."devices":
[			    [
                {"name":"QPU1", "class":"QLM_Default_QPU", "args":{"qpu":"default"}, "module":"tqpm.devices.QLM_Default_QPU"},
                {"name":"QPU2", "class":"QLM_Default_QPU", "args":{"qpu":"LinAlg"}, "module":"tqpm.devices.QLM_Default_QPU"}
]
...
This currently does not work for two reasons:
 _get_instance_with_sub_options does not use 'name' to distinguish between the two diffeernt devices and second the device constructor does not get the device name it requires. This pull request contains a fix for both of these problems.

### CODE REVIEW

File: src/BenchmarkManager.py
Comment (Marvmann) [inline src/BenchmarkManager.py:83]: I agree, that this replaces the if statement above in a more efficient manner. Should be kept

File: src/BenchmarkManager.py
Comment (Marvmann) [inline src/BenchmarkManager.py:75]: Should probably also be "QUARK component" instead of "class"

**Review by Marvmann — COMMENTED** [review]
I added a couple of comments. Overall, I agree with your proposed solution. Would like to wait for the review of @philross before merge, though. Update on Monday.

**Review by philross — APPROVED** [review]

**Pull request merged**
